src/app.py

```diff
@@ -58,7 +59,7 @@ def get_audio_for_each_sentence(sentence_emotion_list, gpt_sovits_client, CHARAC
         # save audio to audio directory
         audio_file_path = save_audio(audio, AUDIO_PATH, CHARACTER_NAME, sentence_emotion['sentence'])
         # save audio file path to sentence_emotion
-        sentence_emotion['audio_file_path'] = f"./{audio_file_path}" # TODO: need to change the path to absolute path
+        sentence_emotion['audio_file_path'] = audio_file_path
```

Comment:
(non-blocking)
Just curious, will `save_audio` return the absolute path if `AUDIO_PATH` is absolute path and will return relative if `AUDIO_PATH` is relative?

Reply:
Yes, I assume `AUDIO_PATH` in .env is absolute path.